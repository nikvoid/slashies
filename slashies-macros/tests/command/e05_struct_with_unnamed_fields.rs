use slashies_macros::Command;

/// An invalid command with unnamed fields
#[derive(Command)]
#[name = "BadCommand"]
struct BadCommand(u64, u64);

fn main() {}
