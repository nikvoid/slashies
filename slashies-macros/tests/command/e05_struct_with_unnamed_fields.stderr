error: Can only derive Command for unit structs or structs with named fields
 --> tests/command/e05_struct_with_unnamed_fields.rs:6:18
  |
6 | struct BadCommand(u64, u64);
  |                  ^^^^^^^^^^
